packages/@react-aria/menu/src/useMenu.ts

```diff
@@ -30,7 +30,10 @@ export interface AriaMenuOptions<T> extends Omit<AriaMenuProps<T>, 'children'> {
    * An optional keyboard delegate implementation for type to select,
    * to override the default.
    */
-  keyboardDelegate?: KeyboardDelegate
+  keyboardDelegate?: KeyboardDelegate,
+  // TODO: Should we have an accompanying onKeyUp even though we don't use it and then just extend KeyboardEvents?
+  // Should this be UNSTABLE? That would mean the props from useSubMenuTrigger would be the UNSTABLE variants as well
+  onKeyDown?: (e: KeyboardEvent) => void
```

Comment:
Open question, this onKeyDown is provided by `useSubMenuTrigger` but wondering if it is weird just to have it by itself without a `onKeyUp` handler. Also would be good to decide what pattern we want for potentially unstable props when they are provided by alpha level hooks. Feels nice to have an alpha hook return the standard DOMAttributes but open to appending UNSTABLE to them

Reply:
Don't add UNSTABLE_